docs/source/visualization/interval_color_mapping.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # All colormaps can be found here: https://matplotlib.org/3.1.0/tutorials/colors/colormaps.html
</pre><p>Readily usable premade colormaps.  Technically users do not have to use only these but it makes it easier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>